src/chainstate/stacks/index/marf.rs

```diff
@@ -599,13 +648,43 @@ impl MARF {
     /// Get open chain tip
     pub fn get_open_chain_tip(&self) -> Option<&BlockHeaderHash> {
         self.open_chain_tip.as_ref()
+            .map(|x| &x.block_hash)
     }
 
     /// Get all known chain tips
     pub fn chain_tips(&mut self) -> Vec<BlockHeaderHash> {
         self.storage.chain_tips()
     }
 
+    /// Check if a block can open successfully, i.e.,
+    ///   it's a known block, the storage system isn't issueing IOErrors, _and_ it's in the same fork
+    ///   as the current block
+    /// The MARF _must_ be open to a valid block for this check to be evaluated.
+    pub fn check_block_hash(&mut self, bhh: &BlockHeaderHash) -> Result<(), Error> {
```

Comment:
Would it be correct to say that this method succeeds only if `bhh` is an ancestor of the currently-open block?  i.e. it's in the same fork history as the currently-open block?  If so, might I suggest a more descriptive name, like `is_ancestor_block_hash()` or the like?

Reply:
Yep, that's right, I can rename.